### DISCUSSION TITLE
Level Designer Toolkit in Sonic Worlds Next

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#148] [Q&A]
How do I use the Level Designer Toolkit in Sonic Worlds Next?

**Comment by r6915ee** [comment]
If you were referring to editing and adding levels, then it's just about the same for Godot in general.

**Comment by DimensionWarped** [comment]
He's referring to this very Zelda/Metroid-vania style tool for level creation and map management.

https://www.youtube.com/watch?v=EJym1soK18E

There appears to be Godot importer around, but I'm not really sure how you'd get the scene transitions working. Honestly though I don't see myself using something like this since the built in level editor is already pretty good. Still, might be a good tool for blocking out a Metroid-Vania style map.

**Comment by r6915ee** [comment]
Ah, alright. If that's the case, then a majority of it has some pretty similar procedures to how Godot does it normally.

**Comment by NeonSRB2** [comment]
This tool seems great for tile variation, but it's probably no better using it over godot's tilemap features, especially since it doesn't seem to support non-square collisions at all, something absolutely essential in sonic levels.

You can use much larger tiles to make a level out of chunks, but you have to make the collisions yourself, though they seem a lot easier to make in godot 4.
